data/forms/residential-status.json

```diff
@@ -1,15 +1,20 @@
 {
   "heading": "Residential status",
   "eligibility": [
+    {
+      "field": "residential-status",
+      "is": "yes",
+      "reasoning": "You have not been continuously residing in Hackney for the last 3 years and you do not fall within any of the listed exceptions"
```

Comment:
This one looks to be a duplicate of the below? and will match if the resident lives in Hackney?